android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -62,10 +59,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         Utils.registerIfUrlValid(swipeRefreshLayout, this, this::refresh);
         setUpRecyclerView();
 
-        realmResults = realmRepo.getSponsors();
-        realmResults.addChangeListener((sponsors, orderedCollectionChangeSet) -> {
-            this.sponsors.clear();
-            this.sponsors.addAll(sponsors);
+        //set up view model
+        SponsorsFragmentViewModel sponsorsFragmentViewModel;
```

Comment:
Please use same line for declaration and assignment

Reply:
@iamareebjamal I am really sorry while rebasing my whole branch got messed up.
So I had to create a new PR for this https://github.com/fossasia/open-event-android/pull/2128
All the requested changes are in this PR
Thank you